core/async_job/async_job.ex

```diff
@@ -108,7 +108,7 @@ defmodule AntikytheraCore.AsyncJob do
       attempts:           attempts,
       remaining_attempts: attempts,
       retry_interval:     Keyword.get(options, :retry_interval, RetryInterval.default()),
-      payload:            payload,
+      payload:            :erlang.term_to_binary(payload),
```

Comment:
Compressing payload at the same time could be beneficial in terms of memory space occupied by the replicated job queue.
Let's add `[:compressed]` as the 2nd argument.
http://erlang.org/doc/man/erlang.html#term_to_binary-2

Reply:
added in 16e243cf7f137a3adf860871e90a12f8a6454133